src/api/database/common.rst

```diff
@@ -55,6 +55,12 @@
                      - :mimetype:`text/plain`
     :>header Content-Type: - :mimetype:`application/json`
                            - :mimetype:`text/plain; charset=utf-8`
+    :>json number cluster.n: Replicas. The number of copies of every document.
+    :>json number cluster.q: Shards. The number of range partitions.
+    :>json number cluster.r: Read quorum. The number of consistent copies
+      of a document that need to be read before successfull reply.
+    :>json number cluster.w: Write quorum. The number of copies of a document
+      need to be written before successfull reply.
```

Comment:
successfull -> successful

Reply:
Maybe more clear "The number of copies of a document that need to be written before successful reply"?